app/scripts/views/mixins/signup-mixin.js

```diff
@@ -31,6 +31,8 @@ define(function (require, exports, module) {
      */
     signUp: function (account, password) {
       var self = this;
+      self.logEvent('flow.signup.submit');
```

Comment:
i think we talked irc about that but then the flow would be:

`begin -> signup.engage -> signin.submit -> account.login`